package.json

```diff
@@ -12,11 +12,11 @@
     "@babel/runtime": "7.8.4",
     "@patternfly/react-charts": "5.2.21",
     "@patternfly/react-core": "3.136.3",
-    "@patternfly/react-icons": "3.14.39",
+    "@patternfly/react-icons": "3.14.41",
     "@patternfly/react-table": "2.26.5",
     "@patternfly/react-tokens": "2.7.25",
-    "@redhat-cloud-services/frontend-components": "0.0.47-beta",
-    "@redhat-cloud-services/frontend-components-inventory-insights": "0.4.2-beta",
+    "@redhat-cloud-services/frontend-components": "1.0.0",
+    "@redhat-cloud-services/frontend-components-inventory-insights": "1.0.0",
     "@redhat-cloud-services/frontend-components-notifications": "0.0.5-RC2",
```

Comment:
These can be changed to 1.0.0 as well I think. We would not want RC versions.

Reply:
FIXITFIXITFIXITFIXIT will hold off until 🤗 🙇‍♀

Reply:
@AllenBW version 1 for the rest of packages should be released momentarily